__tests__/helpers/get-gha-setup.ts

```diff
@@ -33,8 +33,8 @@ export function before(writeFileSyncCb) {
 
   process.env.TEST_CREATEGHA = 'true';
 
-  const spy = jest.spyOn(createGHAObject, 'getPkgVersion');
-  spy.mockReturnValue('7.8.9');
+  const spy = jest.spyOn(getPkgVersion, 'getPkgVersion');
+  spy.mockReturnValue(Promise.resolve('7.8.9'));
```

Comment:
why promiseize this?

Reply:
We have to since we're making a fetch call now 😞

Reply:
oh i missed that at the bottom of this pr